akka-stream/src/main/scala/akka/stream/impl/ActorFlowMaterializerImpl.scala

```diff
@@ -284,4 +284,10 @@ private[akka] object ActorProcessorFactory {
     impl ! ExposedPublisher(p.asInstanceOf[ActorPublisher[Any]])
     p
   }
+
+  def supportingMultipleSubscribers[I, O](impl: ActorRef): ActorProcessor[I, O] = {
+    val p = new ActorProcessor[I, O](impl) with SupportingMultipleSubscribers
+    impl ! ExposedPublisher(p.asInstanceOf[ActorPublisher[Any]])
```

Comment:
wouldn't hurt to have the `// Resolve cyclic dependency with actor. This MUST be the first message no matter what.` here as well I guess